concordia/templates/transcriptions/asset_detail.html

```diff
@@ -306,14 +308,19 @@ <h2>Tags</h2>
                 <span aria-hidden="true">&times;</span>
             </button>
             <div>
-                <ul>
-                    <li>Transcribe original spelling, punctuation, word order, and any page numbers or catalog marks.</li>
-                    <li>Preserve line breaks except when a word breaks over a line or page. Then transcribe it on the line or page where it starts.</li>
-                    <li>Use brackets [ ] around deleted, illegible or partially legible text, and square brackets and asterisks around text in margins [ * * ].</li>
-                    <li>Transcribe any words or letters you can't identify as [?].</li>
-                    <li>Include insertions where you would read them in the text.</li>
-                </ul>
-                Find more detailed instructions in the <a href="{% url 'help-center' %}">Help Center</a>
+                {% if SIMPLE_CONTENT_BLOCKS %}
+                    {% simple_content_block "classic_quicktips" as content_block%}
```

Comment:
```suggestion
                    {% simple_content_block "classic_quicktips" as content_block %}
```

Reply:
Replacing the existing behaviour will also require rendering embedded Django template tags such as the `{% url %}` link to the help center.

I wrote a [`render_inline`](https://django-bittersweet.readthedocs.io/en/latest/quickstart.html#tag-render-inline) tag for this awhile back in `django-bittersweet` which we've already installed so we should be able to either change this to something like:

```
{% render_inline %}{% simple_content_block "classic_quicktips" as content_block %}{% end_render_inline %}
```

… but that's ungainly and it might be preferable to change the `simple_content_block` tag to either extend that or perform the same operation.